### PR TITLE
chore: merge releases back to master

### PR DESCRIPTION
**Pull request by jennijuju**
## Checklist

Before you mark the PR ready for review, please make sure that:
- [ ] All commits have a clear commit message.
- [ ] The PR title is in the form of of `<PR type>: <area>: <change being made>`
    - example: ` fix: mempool: Introduce a cache for valid signatures`
    - `PR type`: _fix_, _feat_, _INTERFACE BREAKING CHANGE_, _CONSENSUS BREAKING_, _build_, _chore_, _ci_, _docs_,_perf_, _refactor_, _revert_, _style_, _test_
    - `area`: _api_, _chain_, _state_, _vm_, _data transfer_, _market_, _mempool_, _message_, _block production_, _multisig_, _networking_, _paychan_, _proving_, _sealing_, _wallet_, _deps_
- [ ] This PR has tests for new functionality or change in behaviour
- [ ] If new user-facing features are introduced, clear usage guidelines and / or documentation updates should be included in https://lotus.filecoin.io or [Discussion Tutorials.](https://github.com/filecoin-project/lotus/discussions/categories/tutorials)
- [ ] CI is green

### CODE REVIEW

File: extern/sector-storage/stores/index.go [hunk 1]
```diff
@@ -25,6 +26,62 @@ import (
...
+	ID     ID
+	URLs   []string // TODO: Support non-http transports
+	Weight uint64
+
+	CanSeal  bool
+	CanStore bool
+
+	Primary bool
+}
+
```
Comment (magik6k) [inline extern/sector-storage/stores/index.go:84]: This was moved to a different package
```suggestion
```

File: extern/storage-sealing/sealing.go [hunk 2]
```diff
@@ -107,6 +108,8 @@ type Sealing struct {
 	assignedPieces map[abi.SectorID][]cid.Cid
 	nextDealSector *abi.SectorNumber // used to prevent a race where we could create a new sector more than once
 
+	upgradeLk sync.Mutex
+	toUpgrade map[abi.SectorNumber]struct{}
```
Comment (magik6k) [inline extern/storage-sealing/sealing.go:112]: This should be removed

File: extern/storage-sealing/upgrade_queue.go [hunk 3]
```diff
@@ -56,3 +59,60 @@ func (m *Sealing) sectorActive(ctx context.Context, tok TipSetToken, sector abi.
...
+		// this one looks good
+		/*if checks */
+		{
+			delete(m.toUpgrade, number)
+			return &number
+		}
+	}
+
+	return nil
+}
```
Comment (magik6k) [inline extern/storage-sealing/upgrade_queue.go:118]: This should be unused?

```suggestion
```

File: extern/storage-sealing/sealing.go (hunk 2 above)
Comment (jennijuju) [reply extern/storage-sealing/sealing.go]: updated

**Review by magik6k — APPROVED** [review]

**Pull request merged**